CHANGELOG.md

```diff
@@ -21,3 +21,11 @@ Given a version number MAJOR.MINOR.PATCH, increment:
 - PixBalance resource for Indirect and Direct Participants
 - PixStatement resource for Direct Participants
 - Event resource for webhook receptions
+- IssuingAuthorization resource 
```

Comment:
Deixar padronizado com o sdk-python

IssuingAuthorization resource for Sub Issuers
IssuingBalance resource for Sub Issuers
IssuingBin resource for Sub Issuers
IssuingCard resource for Sub Issuers
IssuingHolder resource for Sub Issuers
IssuingInvoice resource for Sub Issuers
IssuingPurchase resource for Sub Issuers
IssuingTransaction resource for Sub Issuers
IssuingWithdrawal resource for Sub Issuers